src/components/Sum.jsx

```diff
@@ -1 +1,7 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
+  return (
+    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+  );
+}
```

Comment:
use PropTypes for Sum component
Also, use arrow function for components

Reply:
Okay👌